app/src/main/java/com/mstudio/superheroarch/presentation/detail/CharacterDetailActivity.kt

```diff
@@ -30,7 +31,8 @@ class CharacterDetailActivity : AppCompatActivity(), CharacterDetailViewTranslat
                 TheMovieDbRepository(TheMovieDbApiHelper.create())
             ),
             Dispatchers.IO,
-            NetworkManagerImpl()
+            NetworkManagerImpl(),
+            UpdateFavoriteCharacterStatusUseCase(RickAndMortyRepository(DatabaseHelper.create(), RickAndMortyApiHelper.create()))
```

Comment:
DatabaseHelper.create() and RickAndMortyApiHelper.create() were already called on line 30, better to reuse them.